Arcade/SNAKE/game.ini

```diff
@@ -0,0 +1,5 @@
+title=ASTEROIDS
+date=2018-07-07T07:07:07Z
+author=CDR_Xavier
+description=The snake as it appears on your Nokia screen. Not eating apples.
+license=GNL-2.0
```

Comment:
Likewise, this is also not not right for the same reason.
`GPL-2.0`, not `GNL-2.0`.